komga/src/main/kotlin/org/gotson/komga/infrastructure/mediacontainer/PdfExtractor.kt

```diff
@@ -27,7 +27,7 @@ class PdfExtractor : MediaContainerExtractor {
   private val cache = Caffeine.newBuilder()
     .maximumSize(20)
     .expireAfterAccess(1, TimeUnit.MINUTES)
-    .removalListener { _: Path?, pdf: PDDocument?, _ -> pdf?.close() }
+    .evictionListener { _: Path?, pdf: PDDocument?, _ -> pdf?.close() }
```

Comment:
indeed using `evictionListener` would probably be a better choice, in which case we would also need to change it in `ZipExtractor`

Reply:
Added same thing to ZipExtractor